myuw/test/test_authorization.py

```diff
@@ -0,0 +1,40 @@
+from django.test import TestCase
+from myuw.authorization import validate_netid, INVALID_STRING, NO_USER,\
+    can_override_user, is_myuw_admin
+from myuw.test import get_request_with_user
+
+
+class TestValidation(TestCase):
```

Comment:
This test doesn't pass on my build, could you override the settings required on the individual test case level?

Reply:
https://coveralls.io/builds/16997941

You need to set these in your settings
USERSERVICE_VALIDATION_MODULE = "myuw.authorization.validate_netid"
USERSERVICE_OVERRIDE_AUTH_MODULE = "myuw.authorization.can_override_user"
RESTCLIENTS_ADMIN_AUTH_MODULE = "myuw.authorization.can_proxy_restclient"